src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -102,15 +115,17 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
     public componentDidUpdate(prevProps: Readonly<IVideoAssetProps>) {
         if (this.props.asset !== prevProps.asset) {
             this.setState({ loaded: false });
+        } else {
+            this.addAssetTimelineTags(this.props.childAssets, this.videoPlayer.current.getState().player.duration);
         }
         if (this.props.timestamp !== prevProps.timestamp) {
             this.seekToTimestamp();
         }
     }
 
     /**
-     * Bound to the "Next Tagged Frame" button
-     * Seeks the user to the next tagged video frame
+     * Bound to the "Previous Tagged Frame" button
```

Comment:
Thanks for fixing comments.

Reply:
tbh, I thought they got messed up during my merge, so it was halfway on accident. :)